contributors/chairs-and-techleads/README.md

```diff
@@ -1,36 +1,48 @@
 Home for chairs and tech leads resources and materials for onboarding,
 offboarding, operations, and more.
 
-## Resources for community group leads:    
+## Resources for community group leads: 
 
-### Operations  
+### Operations
 
 In case you are not sure what to include in a meeting invite, here is a [Calendar template](https://github.com/kubernetes/community/blob/master/communication/calendar-guidelines.md#calendar-event-template).
- 
+
 TODO - [Meeting agenda templates]     
 TODO - Crosslink to sig-gov.md, wg-gov,md, gov.md, steering folder in community, etc.
-
+  
 ## Important Governance Documentation
-
-[Code of conduct]  
-- conduct@kubernetes.io if you need anything from them or to report  
-- may have training or guidance - ask!    
-
-[Kubernetes values]  
-[Governance Principles]
-
-### Communication and feedback loops:  
-
-- [Need to Know email archive]  
-- [Monthly meeting]
-
-Slack: #chairs-and-techleads, #sig-contribex  
-Mailing list: leads@kubernetes.io  
-Github: File issues against kubernetes/community  
-
-## Community Programs
-
-- Mentoring, succession planning, and staffing - TODO
+  
+  - [governance.md]
+  - [sig-governanace.md]
+  - [wg-governanace.md]
+  
+  [Code of Conduct]
+  - conduct@kubernetes.io if you need anything from them or to report
+  - may have training or guidance - ask!
+  
+  [Kubernetes values]  
+  [Governance Principles]
+  
+  ## Communication and feedback loops:
+  
+  - [Need to know email archive]
+  - [Monthly meeting]
+  
+  - Time:
+    * 2nd Tuesday of the Month 12:30pm PT [830pmUTC]
+    * 2nd Thursday of the Month 09:00am PT [5pmUTC]
```

Comment:
qq - Are the 8:30 and 5pm meant to link to a timezone converter?

Reply:
It is not linked to the timezone converter but if you will say I will add it.

Reply:
kk, I saw the `[ ]` which usually surrounds a markdown link.^^;;;

Reply:
Okay, So what should I do , should I remove it !

Reply:
Sorry, thought I approved this the other day.